src/plugins/point_in_time_management/public/management_app/mount_management_section.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { I18nProvider } from '@osd/i18n/react';
+import { ManagementAppMountParams } from '../../../management/public';
+import { PointInTimeManagementStartDependencies } from '../plugin';
+import { StartServicesAccessor } from '../../../../core/public';
+import { PointInTimeManagementContext } from '../types';
+import { OpenSearchDashboardsContextProvider } from '../../../opensearch_dashboards_react/public';
+import { EmptyState } from '../components/empty_state';

```

Comment:
nit - it's preferable to import from the `../components` level.

Reply:
done. Thanks